runelite-client/src/main/java/net/runelite/client/plugins/kourendlibrary/KourendLibraryPanel.java

```diff
@@ -55,13 +56,19 @@
 import net.runelite.client.util.SwingUtil;
 
 @Singleton
-public class KourendLibraryPanel extends PluginPanel
+class KourendLibraryPanel extends PluginPanel
 {
-	private static final ImageIcon RESET_ICON;
-	private static final ImageIcon RESET_CLICK_ICON;
+	private Library library;
+	private final KourendLibraryConfig config;
+
+	private static final ImageIcon RESET_ICON, RESET_CLICK_ICON;
 
 	@Inject
-	private Library library;
+	KourendLibraryPanel(Library library, KourendLibraryConfig config)
```

Comment:
If your not doing anything in the ctor just mark the fields @Inject like they were before